README.md

```diff
@@ -47,6 +47,8 @@ Runtime developers can build runtime implementations that run OCI-compliant bund
 
 There is a loose [Road Map](./ROADMAP.md).
 During the `0.x` series of OCI releases we make no backwards compatibility guarantees and intend to break the schema during this series.
+Current and previous releases of the spec in HTML and PDF formats can be found in [Releases](./releases).
+Release notes summarizing key changes are also included here.
```

Comment:
On Wed, Apr 27, 2016 at 04:14:58PM -0700, Rob Dolin (MSFT) wrote:
> > +Current and previous releases of the spec in HTML and PDF formats can be found in [Releases](./releases).
> > +Release notes summarizing key changes are also included here.
> 
> I've updated "here" --> "there"
> IMHO, I think the ChangeLog may be too much detail for folks just looking for a printable version of the spec.  

Right, which is why I like the “Current and previous releases…” line
just the way it is.  Folks interested in HTML and PDF formats *should*
be going to the GitHub release page.

I just think folks who are interested in release notes should be going
to the local ChangeLog (which seems to contain exactly the same
information that is included on the release page, compare [1,2]).

[1]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/ChangeLog#L3-L32
[2]: https://github.com/opencontainers/runtime-spec/releases/tag/v0.5.0